### ISSUE TITLE
InvalidOperationException in SaeaLayerCallback

### ISSUE THREAD

**Issue by thelalle** [Bobris/Nowin#11]
I got this exception and it caused my app to fail.

Exception Info: System.InvalidOperationException
Stack:
   at Nowin.SaeaLayerCallback.StartReceive(Byte[], Int32, Int32)
   at System.Net.Sockets.SocketAsyncEventArgs.FinishOperationSuccess(System.Net.Sockets.SocketError, Int32, System.Net.Sockets.SocketFlags)
   at System.Net.Sockets.SocketAsyncEventArgs.CompletionPortCallback(UInt32, UInt32, System.Threading.NativeOverlapped*)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32, UInt32, System.Threading.NativeOverlapped*)

**Comment by marius-klimantavicius** [comment]
For last two commits (delayed accept) I seem to be unable to find code that actually sets this (DelayedAccept) flag in saea.

**Comment by Bobris** [comment]
Good catch, thanks a lot. It should be in ProcessDisconnect(). I will fix it later.
